From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Fri, 24 Feb 2023 10:20:37 +0100
Subject: [PATCH] Drop workaround with android autofill in compatibility mode

This causes the autofill popup to show when url is tapped. Drop in favor
of native android autofill support compatibility with other autofill
implementations.
---
 .../src/org/chromium/chrome/browser/omnibox/UrlBar.java   | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/UrlBar.java b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/UrlBar.java
index a1103fa7ad0a5..8b2bd89bfb008 100644
--- a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/UrlBar.java
+++ b/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/UrlBar.java
@@ -442,10 +442,10 @@ public class UrlBar extends AutocompleteEditText {
             // the domain changes. We restore this behavior by mimicking the relevant part of
             // TextView.notifyListeningManagersAfterTextChanged().
             // https://cs.android.com/android/platform/superproject/+/5d123b67756dffcfdebdb936ab2de2b29c799321:frameworks/base/core/java/android/widget/TextView.java;l=10618;drc=master;bpv=0
-            final AutofillManager afm = getContext().getSystemService(AutofillManager.class);
-            if (afm != null) {
-                afm.notifyValueChanged(this);
-            }
+            // final AutofillManager afm = getContext().getSystemService(AutofillManager.class);
+            // if (afm != null) {
+            //     afm.notifyValueChanged(this);
+            // }
         }
 
         limitDisplayableLength();
